pyActigraphy/io/bba/bba.py

```diff
@@ -4,8 +4,39 @@
 import re
 
 from ..base import BaseRaw
-from accelerometer.utils import date_parser
-from accelerometer.summarisation import imputeMissing
+from pandas.tseries.frequencies import to_offset
+#from accelerometer.utils import date_parser
+#from accelerometer.summarisation import imputeMissing
+
+# Native implementation of date_parser and imputeMissing
+def date_parser(t):
+    """
+    Adapted from the accelerometer package (utils.py)
+    """
+    tz = re.search(r'(?<=\[).+?(?=\])', t)
+    if tz is not None:
+        tz = tz.group()
+    t = re.sub(r'\[(.*?)\]', '', t)
+    return pd.to_datetime(t, utc=True).tz_convert(tz)
+
+def imputeMissing(data, extrapolate=True):
+    """
+    Adapted from the accelerometer package (summarisation.py).
```

Comment:
Could you:
- provide a link to the accelerometer package? And the version you used (for future ref)? A tag is fine (for ex: v7.2.1);
- add a description for the function as well as for its input parameters (cf other functions in this class)

Reply:
Done!